tool/tctl/common/collection.go

```diff
@@ -477,3 +477,30 @@ func (a *appCollection) toMarshal() interface{} {
 func (a *appCollection) writeYAML(w io.Writer) error {
 	return utils.WriteYAML(w, a.toMarshal())
 }
+
+type authPrefCollection struct {
+	authPrefs []services.AuthPreference
+}
+
+func (c *authPrefCollection) resources() (r []services.Resource) {
+	for _, resource := range c.authPrefs {
+		r = append(r, resource)
+	}
+	return r
+}
+
+func (c *authPrefCollection) writeText(w io.Writer) error {
+	t := asciitable.MakeTable([]string{"Type", "Second Factor", "U2F"})
```

Comment:
nit: If we're expanding the abbreviations, then I'd expect this to look like:
```
Type  | Second Factor | Universal Second Factor
```
?

Reply:
Not sure, U2F might be one of those acronyms that are more understandable when not expanded. And even then, it should perhaps better say "Universal 2nd Factor". Could you weigh in please, @benarent or @webvictim?

Reply:
I'd drop this column entirely.
the U2F app ID is only relevant when `Type == "local"` and `Second Factor == "u2f"`.
if you don't have that specific combo, the column is just confusing.

Reply:
I'll remove the column then.